docs/MVS_Outputs.rst

```diff
@@ -265,20 +265,18 @@ or an excess sink.
 
         &OEM \epsilon \text{[0,1]}
 
+.. _kpi_degree_of_autonomy:
 
 Degree of autonomy (DA)
 #######################
 
-The degree of autonomy describes the relation of the total locally
-generated energy to the total demand of the system (see `[2] <https://www.iip.kit.edu/downloads/McKennaetal_paper_full.pdf>`__).
+The degree of autonomy describes the overall energy consumed minus the energy consumed from the grid divided by the overall energy consumed. Adapted from this definition `[2] <https://pvspeicher.htw-berlin.de/solarspeicherstudie/>`__.
```

Comment:
```suggestion
The degree of autonomy describes the overall energy consumed minus the energy consumed from the grid divided by the overall energy consumed. Adapted from this definition `[3] <https://pvspeicher.htw-berlin.de/solarspeicherstudie/>`__.
```
ahh just remembered: this should be [3] as it is different from the above reference 2 in OEM section